packages/clarity-js/src/layout/mutation.ts

```diff
@@ -218,27 +218,27 @@ async function processNodeList(list: NodeList, source: Source, timer: Timer): Pr
   }
 }
 
-function schedule(node: Node): Node {
+export function schedule(node: Node, fragment: boolean = false): Node {
```

Comment:
>export [](http://example.com/codeflow?start=0&length=6)

Follow up to comment in dom - is it possible to not export it and call it internally?